lib/modules/manager/bazel/parser.spec.ts

```diff
@@ -44,6 +44,10 @@ describe('modules/manager/bazel/parser', () => {
         },
       ],
     });
+    expect(extract(res!)).toMatchObject([
```

Comment:
should we add Todo because of the null override ?

Reply:
Will be fixed right in the next upcoming PR